bucket/clash-for-windows.json

```diff
@@ -0,0 +1,19 @@
+{
+    "homepage": "https://docs.cfw.lbyczf.com",
+    "description": "A Windows GUI based on Clash.",
+    "license": "custom",
```

Comment:
If there is no license, please use `Unknown` . Or provide a url.

Reply:
thank, I have change the license  to "Unknown"